include/arpa/inet.h

```diff
@@ -31,8 +31,22 @@
 #ifndef _ARPA_INET_H
 #define _ARPA_INET_H
 
+#ifndef __OPT_i__
```

Comment:
`#if (__OPT_i__ < 200)`

`__OPT_i__` contains the argument to `--codesize` or just 200 if `-Oi` was given.